src/uct/ib/rdmacm/rdmacm_iface.c

```diff
@@ -594,12 +598,9 @@ static UCS_CLASS_CLEANUP_FUNC(uct_rdmacm_iface_t)
 
     UCS_ASYNC_BLOCK(self->super.worker->async);
 
-    while (!ucs_list_is_empty(&self->used_cm_ids_list)) {
-        cm_id_ctx = ucs_list_extract_head(&self->used_cm_ids_list,
-                                          uct_rdmacm_ctx_t, list);
-        rdma_destroy_id(cm_id_ctx->cm_id);
-        ucs_free(cm_id_ctx);
-        self->cm_id_quota++;
+    ucs_list_for_each_safe(cm_id_ctx, tmp_cm_id_ctx,
+                           &self->used_cm_ids_list, list) {
+        uct_rdmacm_iface_release_cm_id(self, &cm_id_ctx);
```

Comment:
self->cm_id_quota is not updated here... is it ok?

Reply:
`self->cm_id_quota` is update inside `uct_rdmacm_iface_release_cm_id` function
https://github.com/openucx/ucx/blob/f7591208877a19470bcd6b750fbf297147b44eee/src/uct/ib/rdmacm/rdmacm_iface.c#L249